src/database/migrate.ts

```diff
@@ -346,7 +396,7 @@ async function refreshMigrationPosition(
   if (shouldUpdateMigrations) {
     await client.query(`BEGIN`)
     try {
-      await client.query(`DELETE FROM ${migrationTableName}`)
+      await client.query(`DELETE FROM ${migrationTableName} WHERE id is not NULL`)
```

Comment:
running deletes without where statements seems to be blocked by default on staging / prod with a pg extension